compact/ecal/backward_PbWO4.xml

```diff
@@ -39,11 +49,135 @@
             r12max="EcalEndcapN_structure_ring_max"
             SFlength="EcalEndcapN_structure_ring_length"
             CMlength="EcalEndcapN_CrystalModule_length + 2.*EcalEndcapN_CrystalModule_wrap_thickness"
+            NEEMC_PR="EcalEndcapN_Protate"
+            NEEMC_NR="EcalEndcapN_Nrotate"
+            NEEMC_OR_RS="EcalEndcapN_Oring_relative_shift"
+            Inner_a="EcalEndcapN_Iring_axisA"
+            Inner_b="EcalEndcapN_Iring_axisB"
             envelope="true"
-            vis_struc="RPVis"
-            vis_steel_gap="AnlGray"
             ring_material="StainlessSteel"
-            sector="1">
+            sector="1"
+            vis_struc="RPVis"
+            vis_steel_gap="AnlGray">
+
+          <comment>
+            The position information[cm] of inner supporting structure [extrudedpolygon shape, 12 points in total]
```

Comment:
You shouldn't specify the dimensions in a comment. Specify it explicitly in the value, eg `12*cm`.

Reply:
done